packages/edit-site/src/components/header/index.js

```diff
@@ -71,10 +75,16 @@ export default function Header( { openEntitiesSavedStates } ) {
 				<FullscreenModeClose />
 			</MainDashboardButton.Slot>
 			<div className="edit-site-header__toolbar">
-				<Inserter
-					position="bottom right"
-					showInserterHelpPanel
-					toggleProps={ inserterToggleProps }
+				<Button
+					className="edit-post-header-toolbar__inserter-toggle"
```

Comment:
We need to have an `edit-site` class here. I wonder if it makes sense for these styles to live in a shared package instead?

Reply:
What package. At the moment I'm not even certain if at then end of the day, we'd just use edit-post with extensions as edit-site. So I'd prefer to avoid early abstractions like that.

Reply:
`interface`?